paper.tex

```diff
@@ -631,7 +635,7 @@ \subsection{GPU: Execution Time} \label{subsection:gpu-results}
 
 \begin{figure}
     \includegraphics[scale=0.69]{nehab_gpu_timings}
-    \caption{Mean GPU execution times of the compute shader that performs stroke expansion and flattening when run on the \citet{Nehab2020} \emph{timings} data set. The timings are shown in milliseconds. The hatched bars show the timings for the version that outputs line primitives while the solid bars show that for arcs. Mali-G78 (the lowest-end GPU we tested) converts \emph{waves} (the largest scene in the set) to lines in \~3~ms. The compute shader executes at least an order of magnitude faster on the other GPUs.}
+    \caption{Mean GPU execution times of the compute shader that performs stroke expansion and flattening when run on the \citet{Nehab2020} \emph{timings} data set. The timings are shown in milliseconds. The hatched bars show the timings for the version that outputs line primitives while the solid bars show that for arcs. Mali-G78 (the lowest-end GPU we tested) converts \emph{waves} (the largest scene in the set) to lines in about ms. The compute shader executes at least an order of magnitude faster on the other GPUs.}
```

Comment:
the "3" is missing now. Also it looks like I forgot to fix this one up in #33 . I would change this to `\textasciitilde3~ms` which should format correctly.